target/linux/generic/pending-5.15/833-3-dt-bindings-leds-Add-internet-and-rssi-function-defi.patch

```diff
@@ -0,0 +1,31 @@
+From: Shiji Yang <yangshiji66@outlook.com>
+Date: Tue, 31 Oct 2023 17:15:44 +0800
+Subject: [PATCH 3/3] dt-bindings: leds: Add "internet" and "rssi" function
+ definitions
+
+These two types of LEDs are widely used in routers and NICs. The
+RSSI (Received Signal Strength Indicator) LED is used to display
+the Wi-Fi signal strength, and the Internet LED can indicate
+whether the device can access a specific server.
+
+Signed-off-by: Shiji Yang <yangshiji66@outlook.com>
+---
+ include/dt-bindings/leds/common.h | 2 ++
+ 1 file changed, 2 insertions(+)
+
+--- a/include/dt-bindings/leds/common.h
++++ b/include/dt-bindings/leds/common.h
+@@ -82,11 +82,13 @@
+ #define LED_FUNCTION_FLASH "flash"
+ #define LED_FUNCTION_HEARTBEAT "heartbeat"
+ #define LED_FUNCTION_INDICATOR "indicator"
++#define LED_FUNCTION_INTERNET "internet"
```

Comment:
Another function that makes sense IMHO to have is cellular connectivity (LTE, 5G, etc).

Reply:
The patch has already been sent to the mailing list so we need another patch to add it. I think it can be later.

Reply:
Would be nice also get mesh function.

+#define LED_FUNCTION_MESH "mesh"